### DISCUSSION TITLE
3个小建议

### DISCUSSION THREAD

**Discussion by starrynight168** [gkd-kit/gkd#330] [Ideas]
1.增加以root权限保活的功能
2.增加可本地自定义规则，像李跳跳那样可选择应用自定义规则。
3减少应用的内存占用和后台耗电

**Comment by lisonge** [comment]
1. 暂不考虑
2. 自行编辑本地订阅/或者点击默认订阅自己开启想要的规则
3. 内存占用已经优化过了，后台耗电也优化过了，再耗电是你手机问题，或者你也可以提 pr 优化代码，我会考虑合并
4. 如果实在觉得难用可以卸载

**Reply by lisonge** [reply]
补充一下，关于内存占用，我优化过后的占用已经很少了

![img_v3_025l_d5482617-e80e-4693-948a-a1c4a8a5539g](https://github.com/gkd-kit/gkd/assets/38517192/3fae5053-14b5-4e70-a699-f2972c2593fa)

**Reply by starrynight168** [reply]
![Screenshot_2023-11-29-17-26-32-97_f2caa21d3de1d45abee5464dd1c275bf.png](https://github.com/gkd-kit/gkd/assets/99712233/0b0805da-047c-4f3b-a36a-6d956845d2d2)

是的，这个内存占用可以接受，在我手机上平均六七十MB的占用，我觉得内存稍大是相对李跳跳而言的。
